k8s/networks/small.yaml

```diff
@@ -1,6 +1,27 @@
+---
 apiVersion: "keramik.3box.io/v1alpha1"
 kind: Network
 metadata:
   name: small
 spec:
-  replicas: 2
+  replicas: 3
+  datadog:
+    enabled: true
+    version: "testing"
+    profilingEnabled: true
+  ceramic:
+    resourceLimits:
+      storage: 3Gi
+    image: ceramicnetwork/js-ceramic:latest
+    imagePullPolicy: IfNotPresent
+    ipfs:
+      kind: go
+      image: ceramicnetwork/go-ipfs-daemon:latest
+      imagePullPolicy: Always
+
+  # Use custom runner image for bootstrapping
+  bootstrap:
+    image: keramik/runner:dev
+    imagePullPolicy: IfNotPresent
+  cas:
+    image: ceramicnetwork/ceramic-anchor-service:c56be4a3c0bbda012f4b08347e192caf5c405936
```

Comment:
Should this be latest?

Reply:
No latest is broken :( fails to start if it doesn't have AWS creds as it tries to list s3 buckets on startup.

Reply:
I've gotten around this by setting `SQS_QUEUE_URL` to "" in the CAS env.

I don't thins this is comparison is correct https://github.com/ceramicnetwork/ceramic-anchor-service/blob/f0fe6f4a947ec4fffdd48699bae38b8c43112aba/src/services/request-service.ts#L41